src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Nadahar Not sure where you've download it from ?
https://www.ffmpeg.org/releases/ffmpeg-3.3.3.tar.xz
https://ffmpeg.zeranoe.com/builds/